node/src/test/kotlin/net/corda/node/services/network/DBNetworkParametersStorageTest.kt

```
@@ -98,24 +96,11 @@ class DBNetworkParametersStorageTest {
     @Test
     fun `try save parameters with incorrect signature`() {
         database.transaction {
-            val consoleOutput = interceptConsoleOutput {
-                networkParametersService.lookup(hash3)
-            }
-            assertThat(consoleOutput).anySatisfy {
-                it.contains("Caused by: java.security.cert.CertPathValidatorException: subject/issuer name chaining check failed")
-            }
+            // logs a warning (java.security.cert.CertPathValidatorException: Cert path failed to validate)
+            assertNull(networkParametersService.lookup(hash3))
         }
     }
 
-    private fun interceptConsoleOutput(block: () -> Unit): List<String> {
-        val oldOut = System.out
-        val out = ByteOutputStream()
-        System.setOut(PrintStream(out))
-        block()
-        System.setOut(oldOut)
-        return out.bytes.inputStream().bufferedReader().lines().toList()
-    }
```

What doesn't work here?  Is `setOut` gone or something ?  Or is it the `ByteOutputStream` that looks like it was accidentally imported from the wrong place?

---

The test was using an internal class called: `com.sun.xml.internal.messaging.saaj.util.ByteOutputStream` to capture console output as a means of testing the correctness of an internal method (that should return an Exception but returns `null`).